tests/TestUtility/DisposableFile.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.IO;
+
+namespace TestUtility
+{
+    public class DisposableFile : IDisposable
```

Comment:
Is there anything here that can be shared with the code in `TestProject`?

Reply:
good point, I think I will remove this completely, and instead allow a new file to be added to an existing `TestProject`. If the project was shadowcopied, the file will disappear with it, and it if it wasn't we will delete it during the disposal of `TestProject`. This will keep all I/O clean up operations in `TestProject` which is our unit of work in these tests